src/NzbDrone.Core/Datastore/Migration/207_movie_metadata.cs

```diff
@@ -0,0 +1,147 @@
+                .WithColumn("Genres").AsString().Nullable()
+                .WithColumn("Title").AsString()
+                .WithColumn("SortTitle").AsString().Nullable()
+                .WithColumn("CleanTitle").AsString().Indexed()
+                .WithColumn("OriginalTitle").AsString()
+                .WithColumn("CleanOriginalTitle").AsString().Indexed()
+                .WithColumn("OriginalLanguage").AsInt32()
+                .WithColumn("Status").AsInt32()
+                .WithColumn("LastInfoSync").AsDateTime().Nullable()
+                .WithColumn("Runtime").AsInt32()
+                .WithColumn("InCinemas").AsDateTime().Nullable()
+                .WithColumn("PhysicalRelease").AsDateTime().Nullable()
+                .WithColumn("DigitalRelease").AsDateTime().Nullable()
+                .WithColumn("Year").AsInt32().Nullable()
+                .WithColumn("SecondaryYear").AsInt32().Nullable()
+                .WithColumn("Ratings").AsString().Nullable()
+                .WithColumn("Recommendations").AsString().Nullable()
+                .WithColumn("Certification").AsString().Nullable()
+                .WithColumn("YouTubeTrailerId").AsString().Nullable()
+                .WithColumn("Collection").AsString().Nullable()
+                .WithColumn("Studio").AsString().Nullable()
+                .WithColumn("Overview").AsString().Nullable()
+                .WithColumn("Website").AsString().Nullable();
+
+            Execute.Sql(@"INSERT INTO MovieMetadata (TmdbId, ImdbId, Title, SortTitle, CleanTitle, OriginalTitle, CleanOriginalTitle, OriginalLanguage, Overview, Status, LastInfoSync, Images, Genres, Ratings, Runtime, InCinemas, PhysicalRelease, DigitalRelease, Year, SecondaryYear, Recommendations, Certification, YouTubeTrailerId, Studio, Collection, Website)
+                          SELECT TmdbId, ImdbId, Title, SortTitle, CleanTitle, OriginalTitle, CleanTitle, OriginalLanguage, Overview, Status, LastInfoSync, Images, Genres, Ratings, Runtime, InCinemas, PhysicalRelease, DigitalRelease, Year, SecondaryYear, Recommendations, Certification, YouTubeTrailerId, Studio, Collection, Website
+                          FROM Movies");
+
+            Execute.Sql(@"INSERT INTO MovieMetadata (TmdbId, ImdbId, Title, SortTitle, CleanTitle, OriginalTitle, CleanOriginalTitle, OriginalLanguage, Overview, Status, LastInfoSync, Images, Genres, Ratings, Runtime, InCinemas, PhysicalRelease, DigitalRelease, Year, Certification, YouTubeTrailerId, Studio, Collection, Website)
+                          SELECT TmdbId, ImdbId, Title, SortTitle, Title, OriginalTitle, OriginalTitle, 1, Overview, Status, LastInfoSync, Images, Genres, Ratings, Runtime, InCinemas, PhysicalRelease, DigitalRelease, Year, Certification, YouTubeTrailerId, Studio, Collection, Website
+                          FROM ImportListMovies
+                          WHERE ImportListMovies.TmdbId NOT IN ( SELECT MovieMetadata.TmdbId FROM MovieMetadata )");
```

Comment:
Postgres support on these please dude :D (and the various other SQL stuff in this migration)